StopSelector/js/main.js

```diff
@@ -283,7 +282,8 @@ function busBoardURL(stops) {
   
   // Remove the last section of the URL, because the repo is structured with
   // the BusInfoBoard one directory up
-  return url.split("/").slice(0,-1).join("/") + query_string + "&mobile=true&stops=" + stops.join("+");
+  var ret = url.slice(0,-1).join("/") + query_string + "&mobile=true&stops=" + stops.join("+");
```

Comment:
My question is: Why are we calling `.slice()` and `.join()` on a thing that is now a string?